src/main.ts

```diff
@@ -25,6 +25,23 @@ export {
  */
 export { RobotClient as Client } from './robot';
 
+/**
+ * @deprecated Use {@link ViamClient} instead.
+ * @group Clients
+ */
+export { ViamClient } from './app/viam-client';
+
+/**
+ * Raw Protobuf interfaces for a Data service.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link DataClient} instead.
```

Comment:
Is the `DataClient` exported from this package or just the `ViamClient`, which has a public `dataClient` property as an instance of the `DataClient`?